.travis.yml

```diff
@@ -3,8 +3,6 @@ python: 2.7
 env:
     - TOX_ENV=py26
     - TOX_ENV=py27
-    - TOX_ENV=py35
```

Comment:
I don't think we should remove them, however you can probably ignore the async tests for py35 too.

Reply:
I'm very new to Tox and not very familiar with Travis, but from what I understand this is instructing it to run the py35 test and flake8 test in the Python 2.7 test runner. I updated the [Tox config](https://github.com/danjac/pyramid_storage/pull/41/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R17) for pytest to ignore the entire async files when using Py26 or Py27, but those files should not be ignored for py35. There is no Python 3.5 test runner in this setup - the variable seems to be incorrect. I can add a section for Python 3.5, but it wasn't actually being tested before and it's probably not a good idea to add new testing for a deprecated version.

The flake8 test is also being run under the [Python 3.6 test](https://github.com/danjac/pyramid_storage/pull/41/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R22) and that test doesn't need to ignore any files.

In order to keep the flake8 check under this Python 2.7 runner as well, I'll have to get creative with the flake8 setup and convince it to also conditionally ignore the async files. I believe the check is redundant and the easiest course of action is to just remove it.